src/App.tsx

```diff
@@ -1,26 +1,192 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import {
+  addTodo,
+  deleteTodos,
+  getTodos,
+  toggleTodo,
+  USER_ID,
+} from './api/todos';
 
-const USER_ID = 0;
+import { FilterStatus } from './utils/FilterStatus';
+import { Todo } from './types/Todo';
+import { Error } from './components/Error';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { TodoItem } from './components/TodoItem';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [errorMessage, setErrorMessage] = useState<string>('');
```

Comment:
should be enum with error messages
```suggestion
  const [errorMessage, setErrorMessage] = useState<ErrorMessages>(ErrorMessages.Default);
```